managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -146,6 +151,13 @@
 
     protected static final int AsyncOperationTimeoutSeconds = 30;
 
+    protected static final String DELETABLE_LEDGER_MARKER_KEYWORD = "pulsar.ml.deletable.ledgers";
```

Comment:
We will append the prefix for each ledger, and store the property map into ZNode. However, if there are thousands of ledgers to delete, the property map will cost too many storage space especially into Znode, which limit by 5MB. The key point is the prefix will duplicate for each ledger. We'd better redesign the data structure of the properties storage.

Reply:
Thanks for your question. IMO, the problem of over-size znode should theoretically be a discussion about whether `ManagedLedgerInfo` should be stored on zk. Or we can think of it this way, the ledger information stored in the property map actually comes from the `LedgerInfoList` structure, and this part of the data is also stored in a znode. Therefore, if the number of ledgers is large, the corresponding ml-znode must be huge. To avoid complicating matters, I think we can skip this part in this proposal, and start a new discussion thread on this problem.

Reply:
For current implementation, we shouldn't introduce more overload for current managedLedger Znode storage. we'd better reduce the prefix of keyword or redesign the data structure for properties storage.

Reply:
Maybe we keep the `d_ledgers` is enough? The `pulsar.ml` looks useless.